xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -25,8 +25,9 @@
 
 #include "platform/win32/CharsetConverter.h"
 
-// @todo: Remove this and "SDK_26100.h" when Windows SDK updated to 10.0.26100.0 in builders
 #ifdef TARGET_WINDOWS_DESKTOP
+#include "DisplayUtilsWin32.h"
+// @todo: Remove this and "SDK_26100.h" when Windows SDK updated to 10.0.26100.0 in builders
 #ifndef NTDDI_WIN11_GE // Windows SDK 10.0.26100.0 or newer
 #include "SDK_26100.h"
 #endif
```

Comment:
These lines should be moved to DisplayUtilsWin32.cpp

Reply:
yes absolutely